### ISSUE TITLE
Packet clone can't be modified

### ISSUE THREAD

**Issue by Foogeebar** [seladb/PcapPlusPlus#990]
Hi,
I am using pcpp for a home work and i discover a small bug in class Packet in these 2 methods:

```
Packet(const Packet& other) { copyDataFrom(other); }
Packet& operator=(const Packet& other);
```

The member variable `m_CanReallocateData ` should be explicitly initialized as it is done in others constructors in order to permit reallocation in case of packet modification.

Thx for your excellent work.

**Comment by seladb** [comment]
You're right, this is indeed a bug, thank you for reporting it!

Would you mind opening a PR with a fix?

**Comment by seladb** [comment]
> You're right, this is indeed a bug, thank you for reporting it!
> 
> Would you mind opening a PR with a fix?

@Foogeebar I'm not sure if you saw my previous, would you consider opening a PR with a fix?

**Comment by seladb** [comment]
Fixed in #1073 

Thanks again @Foogeebar for reporting this issue!
